lib/pubsub/subscription.js

```diff
@@ -401,7 +401,7 @@ Subscription.prototype.pull = function(options, callback) {
  */
 Subscription.prototype.setAckDeadline = function(options, callback) {
   var body = {
-    ackId: options.ackId,
+    ackIds: util.arrayize(options.ackIds),
```

Comment:
@stephenplusplus You could have just made it `ackIds: util.arrayize(options.ackIds || options.ackId),` to not make it a breaking change.